ClimsoftVer4/ClimsoftVer4/frmImportDaily.vb

```diff
@@ -167,7 +172,13 @@
                                 Scale_Data(code, dat)
                                 'MsgBox(dat)
                             End If
-                            Add_Record(stn, code, datetime, dat)
+<<<<<<< HEAD
+                            If IsDate(datetime) Then If Not Add_Record(stn, code, datetime, dat) Then Exit Sub
+=======
+
+                            If IsDate(datetime) Then Add_Record(stn, code, datetime, dat)
+>>>>>>> origin/beta2
+
```

Comment:
I've just had a look at the latest pull request - the reason that it is failing to merge is because an attempt has been made to fix a conflict - but the conflict resolution was not completed.  This has left places in `frmImportDaily.vb` that have extra text in addition to valid visual basic code that looks like this:

    <<<<<<< HEAD
        If IsDate(datetime) Then If Not Add_Record(stn, code, datetime, dat) Then Exit Sub
    =======
        If IsDate(datetime) Then Add_Record(stn, code, datetime, dat)
    >>>>>>> origin/beta2

The way to solve this is to go through and, in each case, choose which of the two lines shown is correct.  Then keep only this and remove the lines that say `<<< HEAD`, `====` and `>>> origin/beta2`